src/components/HomePage.js

```diff
@@ -0,0 +1,241 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Menu, Modal, Input, Grid, Button, Form } from 'semantic-ui-react';
+
+import Styles from '../styles/HomePage';
+import Actions from '../reducers/Actions'
+import { statekeys } from '../helpers/Common' 
+
+export class HomePage extends Component {
+
+  state = { modalOpenSignUp: false }
+
+  handleOpenSignUp = () => this.setState({ modalOpenSignUp: true })
+
+  handleCloseSignUp = () => this.setState({ modalOpenSignUp: false })
+        
+  render() {
+      const {
+            setEmail,
+            setPassword,
+            setFirstName,
+            setLastName,
+            setPasswordRepeat,
+            setAddress,
+            setCity,
+            setAddressState,
+            setZipcode,
+            openLoginModal,
+            closeLoginModal,
+            isLoginModalOpen,
+            openSignupModal,
+            closeSignupModal,
+            isSignupModalOpen
```

Comment:
Are the signup modal or the login modal different and need separate values in the store?